solvebio/exporters.py

```diff
@@ -200,6 +215,119 @@ def write(self, filename):
             f.close()
 
 
+class FlatCSVExporter(object):
```

Comment:
is it possible to subclass `CSVExporter` and use `__init__` and `write()` from the parent class?